packages/evm/contracts/registry/GuardiansRegistry.sol

```diff
@@ -255,7 +251,7 @@ contract GuardiansRegistry is IGuardiansRegistry, ControlledRecoverable, Signatu
         emit GuardianActivationLockChanged(_guardian, _lockManager, newLockedAmount, newTotalLocked);
 
         if (_requestDeactivation) {
-            _validateSignature(_guardian);
+            _authenticateSender(_guardian);
```

Comment:
Let's leave a comment here about our expected flow (that for the deactivation to happen with the unlock, the request must originate from the owner).